src/index.ts

```diff
@@ -69,7 +69,14 @@ export async function handler(event: IAutoRelayHandler, context?: IRunContext):
    * Future versions will also include an ethers provider aware of this.
    */
 
-  let cUsdPrice = await executeCusdService(coinGeckoApiKey, signer);
+  let cUsdPrice: number | undefined;
```

Comment:
please remove all of these try/catches -- the error handling is already the way I want it and working properly, thanks.  Otherwise please indicate specifically what is demonstrably not working the way you think it should. thanks

Reply:
Done.  The code already has this behavior.  See: https://www.notion.so/curvelabs/Kolektivo-Reserve-Kolektivo-services-using-OpenZeppelin-Defender-f7430652d5684df29ef9197a1798325e?pvs=4#6a7560a1c28b46abadb55c0a4f64dcac